app/Lib/lang.php

```diff
@@ -2186,7 +2193,10 @@
   'op.manage' =>      'Manage',
   'op.manage-a' =>    'Manage %1$s',
   'op.manage.grm' =>  'Manage Group Memberships',
+  'op.match.add.confirm' => 'Are you sure you wish to add a new CO Person record?',
```

Comment:
"Are you sure you want to create a new CO Person record"?

Reply:
Changed as suggested.